themes/digital.gov/src/scss/new/guides/_guide-landing.scss

```diff
@@ -0,0 +1,57 @@
+@use "uswds-core" as *;
+
+.dg-guides {
+  padding-top: units(2);
+
+  @include at-media("tablet") {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @include at-media("desktop") {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+.dg-guides-card {
```

Comment:
Are there any styles here that can be used across cards?

![image](https://github.com/GSA/digitalgov.gov/assets/3385219/993ea249-90ef-4bbd-b7b5-d77424586b4c)

![image](https://github.com/GSA/digitalgov.gov/assets/3385219/1d4f0eb5-48dc-4d9c-a1f0-a0120dfe3613)

![image](https://github.com/GSA/digitalgov.gov/assets/3385219/5ecad0b6-ec8f-447f-ba00-8fe6149e7d5c)

Reply:
@mejiaj Good call. I've updated the guides card grid to match the HCD card grid. This will reuse existing code, address the style comment below and set us up for refactoring this into a more generic card component.